repo2docker/buildpacks/_r_base.py

```diff
@@ -55,8 +55,8 @@ def rstudio_base_scripts():
             "${NB_USER}",
             # Install nbrsessionproxy
             r"""
-                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-server-proxy/archive/7ac0125.zip && \
-                pip install --no-cache-dir jupyter-rsession-proxy==1.0b6 && \
+                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-server-proxy/archive/00be53dfed8ab5a0baaf8332e3fc6bd44dc5bf1a.zip && \
```

Comment:
I've recently made a 1.3.1 release, should we use that?
https://pypi.org/project/jupyter-server-proxy/

Reply:
:+1: If there's a new improvement in that justifies using a commit I think that's a good reason to make a new release of `jupyter-server-proxy` so everyone can use it, not just repo2docker.

Reply:
```suggestion
                pip install --no-cache-dir jupyter-server-proxy==1.3.2 && \
```